File: 백준/Silver/11650. 좌표 정렬하기/좌표 정렬하기.py
import sys

N = int(sys.stdin.readline())
obj = {}
for _ in range(N) :
  x,y = map(int,sys.stdin.readline().split())

  if x not in obj :
    obj[x] = [y]
  else :
    obj[x].append(y)

for nm in sorted(obj):
  obj[nm].sort()
  if len(obj[nm]) > 1 :
    for i in obj[nm] :
      print(f"{nm} {i}")
  else :
    print(f"{nm} {obj[nm][0]}")

File: 백준/Silver/11650. 좌표 정렬하기/좌표 정렬하기.js
const readLine = require('fs').readFileSync('dev/stdin').toString().trim();
const [N, ...input] = readLine.split('\n');

const answer = input.sort((a, b) => {
  const [x1, y1] = a.split(' ').map(Number);
  const [x2, y2] = b.split(' ').map(Number);

  if (x1 === x2) {
    return y1 - y2;
  }
  return x1 - x2;
});

console.log(answer.join('\n'));